aapcs32/aapcs32.rst

```diff
@@ -819,11 +819,11 @@ one status register (CPSR) that is available for use in conforming code.
    +----------+---------+---------+-----------------------------------------+
    | r10      | v7      |         | Variable-register 7.                    |
    +----------+---------+---------+-----------------------------------------+
-   | r9       |         | v6      | Platform register.                      |
-   |          |         |         |                                         |
-   |          |         | SB      | The meaning of this register is defined |
-   |          |         |         | by the platform standard.               |
+   | r9       | v6      | SB      | Platform register or Variable-register  |
+   |          |         |         | 8.                                      |
```

Comment:
The 4th column has Variable-register 8, this should be Variable-register 6 to match v6.

Reply:
Whoops. Well, I just pulled and amended locally, but now it won't let me push. Apparently I'm a commit behind as well as being a commit ahead. I'm really not good at Git it seems.

Reply:
Whoops. Well, forced that down, should be good now.

Reply:
you should be able to just force-push: `git push -uf <repo> <branch>`

Reply:
Yep, did that, thanks. I really should properly learn Git.